soc/microchip/mec/mec1501x/Kconfig

```diff
@@ -3,15 +3,18 @@
 # Copyright (c) 2018 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-choice
-	prompt "MEC1501 Selection"
-	depends on SOC_SERIES_MEC1501X
+config SOC_SERIES_MEC1501X
+	select ARM
+	select CPU_CORTEX_M4
+	select CPU_CORTEX_M_HAS_DWT
+	select CPU_HAS_ARM_MPU
+	select SOC_FAMILY_MEC
```

Comment:
`SOC_FAMILY_*` selection goes in `Kconfig.soc`

Reply:
cool, it was there already, guess I forgot to delete it from here